runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -59,6 +59,31 @@ public final class AnimationID
 	public static final int WOODCUTTING_2H_CRYSTAL = 10072;
 	public static final int WOODCUTTING_2H_CRYSTAL_INACTIVE = 10073;
 	public static final int WOODCUTTING_2H_3A = 10074;
+	/*Ent animation shared with Canoe cutting*/
+	public static final int WOODCUTTING_ENT_BRONZE = 3291;
+	public static final int WOODCUTTING_ENT_IRON = 3290;
+	public static final int WOODCUTTING_ENT_STEEL = 3289;
+	public static final int WOODCUTTING_ENT_BLACK = 3288;
+	public static final int WOODCUTTING_ENT_MITHRIL = 3287;
+	public static final int WOODCUTTING_ENT_ADAMANT = 3286;
+	public static final int WOODCUTTING_ENT_RUNE = 3285;
+	public static final int WOODCUTTING_ENT_DRAGON = 3292;
```

Comment:
`WOODCUTTING_ENT_DRAGON_OR` is missing; from testing with dev tools, it appears to be `23`--it would be helpful to have this confirmed though.

Reply:
I'm not sure what `WOODCUTTING_ENT_INFERNAL` is offhand, I can test later with a real infernal axe if needed.

Reply:
Dragon Axe (or) is 23